.github/workflows/karpenter-comment.yaml

```
@@ -0,0 +1,22 @@
+name: Karpentercomment
```

wdyt about calling it something like this
```suggestion
name: manual-snapshot
```